tests/binary_extensions_test.py

```diff
@@ -27,14 +27,14 @@ def test_success(datadir: Path) -> None:
         '@scope-one/package-three.extension.blue.BlueColorExtension',
     ]
     assert list(extensions.extensions.keys()) == ['color', 'greeting']
-    assert list(extensions.imports.keys()) == [
+    assert list(extensions.imports.keys()).sort() == [
```

Comment:
On the CI server they're loaded in a different order than our machines and it break the tests :) 

lmk if this shouldn't be the case, but I looked at the code and seems ok.